packages/orchestrator-ui-components/src/pages/startPage/WfoStartPage.tsx

```diff
@@ -111,7 +120,27 @@ export const WfoStartPage = () => {
             name: t('activeWorkflows.buttonText'),
             url: PATH_WORKFLOWS,
         },
-        isLoading: processesSummaryIsFetching,
+        isLoading: activeWorkflowsSummaryIsFetching,
+    };
+
+    const myWorkflowsSummaryCard: SummaryCard = {
+        headerTitle: t('myWorkflows.headerTitle'),
+        headerValue: myWorkflowsSummaryResponse?.pageInfo.totalItems ?? 0,
+        headerStatus: SummaryCardStatus.Success,
+        listTitle: t('myWorkflows.listTitle'),
+        listItems:
+            myWorkflowsSummaryResponse?.processes.map((workflow) => ({
```

Comment:
This code looks a lot like the code on line 114. Maybe extract the common functionality in a separate component?

Reply:
Done